base/deprecated.jl

```diff
@@ -1076,4 +1077,6 @@ eval(Base.LinAlg, quote
     end
 end)
 
+Filesystem.stop_watching(stream::Filesystem._FDWatcher) = depwarn("stop_watching(::_FDWatcher) is removed")
```

Comment:
nobody will see this, since nobody was using it, so it doesn't really matter what it says. but as it says, the correct action is to remove the call from the code.

The `stop_watching` in ZMQ is a different function.